assets/css/main.css

```diff
@@ -282,8 +283,35 @@ li {
   list-style-type: none;
 }
 
+textarea.context-of-advice {
+  display: block;
+  margin: 0 auto;
+  width: 50%;
+}
+
 /* ==== PROVIDE ==== */
 
+.message-area {
+  margin: 0px auto;
+  max-width: 600px;
+  text-align: center;
+  padding: 15px;
+
+}
+
+blockquote {
+  background: #ececec;;
+  border-left: 5px solid #888282;
+  margin: 1.5em 0;
+  padding: 0.5em 10px;
+  quotes: "\201C""\201D""\2018""\2019";
```

Comment:
What is that line about?

Reply:
I was surprised mysefl... probably a copy-paste leftover